controlplane/internal/controller/clusterdeployment_controller.go

```diff
@@ -273,3 +285,92 @@ func (r *ClusterDeploymentReconciler) createImageRegistry(ctx context.Context, r
 	}
 	return nil
 }
+
+type InstallConfigOverride struct {
+	Capability Capability `json:"capabilities,omitempty"`
+}
+type Capability struct {
+	BaselineCapability            string   `json:"baselineCapabilitySet,omitempty"`
+	AdditionalEnabledCapabilities []string `json:"additionalEnabledCapabilities,omitempty"`
+}
```

Comment:
Isn't this exactly the same as https://github.com/openshift/installer/blob/74c0c30aa1e7926302b07564480a8eb2f605be20/pkg/types/installconfig.go#L551 ? Shouldn't we use that instead?
It's ok for the OpenshiftAssistedControlPlane spec, because we might be adding additional validation and we can have better control of our API, but this is intended for install config so I think it would make more sense. Would you agree?

Reply:
Sure changed it